.github/workflows/test-matrix.yml

```
@@ -0,0 +1,52 @@
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false # Continue other combos even if one fails
+            matrix:
+                node: [18, 20, 22, 24]
+                react: ["17.0.2", "18.2.0"] # TODO: Add "19.2.1" when ready
+
+        steps:
+            - name: Pack library
+              id: pack
+              run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2035:info:2:20: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)